frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.tsx

```diff
@@ -56,6 +57,8 @@ export const PublicOrEmbeddedQuestion = ({
     {},
   );
 
+  const downloadsEnabled = PLUGIN_DOWNLOADS_ENABLED.areDownloadsEnabled();
```

Comment:
This is where I'll pass down the hash parameters (hide_download_button and downloads) in the next task